packages/api/cms-api/src/blocks/createImageLinkBlock.test.ts

```diff
@@ -0,0 +1,10 @@
+import { InternalLinkBlock } from "../page-tree/blocks/internal-link.block";
+import { createImageLinkBlock } from "./createImageLinkBlock";
+
+describe("createImageLinkBlock", () => {
+    it("should allow overriding the name", () => {
+        const MyCustomImageLinkBlock = createImageLinkBlock({ link: InternalLinkBlock }, "MyCustomImageLink");
+
+        expect(MyCustomImageLinkBlock.name).toEqual("MyCustomImageLink");
+    });
```

Comment:
nit: You could additionally add a "countercheck" of the default behavior to verify that this is actually a change caused by the config

Reply:
Like this? a5112d4a0f501068b0341bf4d8b91a25b7932ca9